themes/wporg-translate-events-2024/blocks/header/render.php

```diff
@@ -0,0 +1,22 @@
+<?php namespace Wporg\TranslationEvents\Theme_2024;
+
+/** @var string $site_header */
+/** @var array $attributes */
+
+$html_title = implode( ' | ', array( $attributes['title'], __( 'Translation Events', 'wporg-translate-events-2024' ) ) );
+
+?><!DOCTYPE html>
+<html <?php language_attributes(); ?>>
+	<head>
+		<meta charset="<?php bloginfo( 'charset' ); ?>" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<title><?php echo esc_html( $html_title ); ?></title>
+		<?php wp_head(); ?>
+	</head>
+	<body <?php body_class(); ?>>
+		<?php wp_body_open(); ?>
+		<div class="wp-site-blocks">
+			<?php // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+			<?php echo $site_header; ?>
```

Comment:
I am not quite sure what the benefit is to use a variable here vs using the outside do_blocks. Then we could just do this:

```suggestion
<?php require __DIR__ . '/site-header.php'; ?>
```

Reply:
I've provided some information on why I think this is needed in https://github.com/WordPress/wporg-gp-translation-events/pull/321#issuecomment-2225359200